add builder cowboying how-to (#168)
canonical/launchpad-manual: developer/how-to/debug-buildfarm-builder.rst

   ``disable`` the builder, otherwise the build might not get dispatched
   properly.

> .. _Disable Builder:

Reviewer: ```suggestion
.. _disable-builder:
```

   ``disable`` the builder, otherwise the build might not get dispatched
   properly.

> .. _Disable Builder:
> Disable a builder

Reviewer: ```suggestion
.. _Disable Builder:

Disable a builder
```

> .. _disable-builder::
> Disable a builder

Reviewer: ```suggestion
.. _disable-builder::

Disable a builder
```

session from within the container.

Cowboy builder
~~~~~~~~~~~~~~

.. tip::
   Either :ref:`disable<Disable Builder>` or switch to manual every
   builder in the same architecture except one, and make the cowboy on that
   builder only. Otherwise, the location of the next triggered build will be
   randomized.

Once you are inside the builder, 

> 1. Go to the package directory of lp-buildd with ``cd 

Reviewer: ```suggestion
1. Switch to the package directory of lp-buildd with ``cd 
```

   /usr/lib/python3/dist-packages/lpbuildd/``.
   

> 2. In there make the changes you want with ``sudo`` access, e.g. ``sudo vim

Reviewer: ```suggestion
2. Make the changes you want with ``sudo`` access, e.g. ``sudo vim
```

   binarypackage.py``.

3. And once the changes are done, restart the launchpad-buildd systemd service
   with ``sudo systemctl restart launchpad-buildd``

> This will restart the service with your changes added. Then running any build
> on that specific builder will run the cowboyed changes with it.

Reviewer: ```suggestion
This will restart the service with your changes included. Running any build
on that specific builder will run the cowboyed changes with it.
```

.. note::

>    The changes/cowboy to the builder will be deleted in the next "cleaning"

Reviewer: ```suggestion
   The cowboyed changes to the builder will be deleted in the next "cleaning"
```

   stage of the builder. Which can be triggered by finished builds or
   disable-then-enable calls. Each subsequent cowboyed build will require

>    re-application of the cowboy in an identical manner.

Reviewer: ```suggestion
   re-application of the cowboy changes in an identical manner.
```
